app/res/values/colors.xml

```diff
@@ -130,4 +130,22 @@
     <color name="cc_error_bg_color">#EE5555</color>
     <color name="cc_error_text_color">#FFFFFF</color>
     <color name="map_layer_button_color">#D3D3D3</color>
+
+    <!--    Map  Buttons-->
+    <color name="gray_300">#E0E0E0</color>
```

Comment:
Does this mean we'll get the ability to restyle the Kujaku views on our side? That would be cool

Reply:
ahh no, I forgot to clean up these colors from the Area mapper port. removed the unused ones.